packages/common/src/urls/getContentHomeUrl.ts

```diff
@@ -0,0 +1,11 @@
+import { IPortal } from "@esri/arcgis-rest-portal/dist/esm/util/get-portal";
+import { IHubRequestOptions } from "../types";
+import { IRequestOptions } from "@esri/arcgis-rest-request/dist/esm/utils/IRequestOptions";
+import { getPortalUrl } from "./get-portal-url";
+
+export function getContentHomeUrl(
```

Comment:
I _believe_ you need to add an export statement in `packages/common/src/urls/index.ts` otherwise you won't be able to import this directly from `@esri/hub-common`